specification/cognitiveservices/resource-manager/readme.md

```diff
@@ -64,7 +64,7 @@ csharp:
   azure-arm: true
   license-header: MICROSOFT_MIT_NO_VERSION
   namespace: Microsoft.Azure.Management.CognitiveServices
-  output-folder: $(csharp-sdks-folder)/CognitiveServices/Management.CognitiveServices/Generated
+  output-folder: $(csharp-sdks-folder)/CognitiveServices/management/Management.CognitiveServices/Generated

```

Comment:
Checkout here: https://github.com/Azure/azure-sdk-for-net/tree/psSdkJson6/src/SDKs/CognitiveServices
now management SDK is under "management" folder so we need this change to make generate.cmd outputs code to the right folder.